test/internals/handleRequest.test.js

```diff
@@ -36,6 +37,14 @@ test('Request object', t => {
   t.strictDeepEqual(req.body, null)
 })
 
+test('handleRequest function - sent reply', t => {
```

Comment:
which line is this testing?

Reply:
This https://github.com/flickz/fastify/blob/139cbf40d1e4b6c0dae26701883071bc9af80439/lib/handleRequest.js#L9